### PR TITLE
KOGITO-7710 : Potential leak of authentication information

### PR DESCRIPTION
**Pull request by Rikkola**
**JIRA**: [KOGITO-7710](https://issues.redhat.com/browse/KOGITO-7710)

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* Retest PR: <b>jenkins retest this</b>
* A full downstream build: <b>jenkins do fdb</b>
* A compile downstream build: <b>jenkins do cdb</b>
* A full production downstream build: <b>jenkins do product fdb</b>
* An upstream build: <b>jenkins do upstream</b>
</details>

### CODE REVIEW

**Comment by domhanak** [comment]
@Rikkola Reasonable change, shall I review and verify?

**Review by domhanak — APPROVED** [review]

**Comment by Rikkola** [comment]
```
Failed to execute goal org.wildfly.plugins:wildfly-maven-plugin:1.2.0.Final:deploy-only (deploy-workbench) on project jbpm-wb-showcase: Failed to execute goal deploy-only: {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-1" => {"WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"jbpm-wb-showcase.war\".WeldStartService" => "Failed to start service
[2022-08-05T09:50:51.665Z] [ERROR]     Caused by: org.jboss.weld.exceptions.DeploymentException: WELD-000049: Unable to invoke void org.kie.workbench.common.screens.archetype.mgmt.backend.service.ArchetypeServiceImpl.postConstruct() on org.kie.workbench.common.screens.archetype.mgmt.backend.service.ArchetypeServiceImpl@4796cbc9
[2022-08-05T09:50:51.665Z] [ERROR]     Caused by: org.jboss.weld.exceptions.WeldException: WELD-000049: Unable to invoke void org.kie.workbench.common.screens.archetype.mgmt.backend.service.ArchetypeServiceImpl.postConstruct() on org.kie.workbench.common.screens.archetype.mgmt.backend.service.ArchetypeServiceImpl@4796cbc9
```

jenkins do fdb

**Comment by Rikkola** [comment]
Jenkins run fdb

**Comment by Rikkola** [comment]
Jenkins run fdb

**Comment by Rikkola** [comment]
Jenkins run fdb

jbpm-webapp issue

**Comment by Rikkola** [comment]
Jenkins run fdb

jbpm-webapp issue

**Comment by Rikkola** [comment]
Jenkins run fdb

**Review by RishiRajAnand — APPROVED** [review]

**Comment by Rikkola** [comment]
@kiegroup/gatekeepers Can I get a merge please.

File: kie-wb-common-ala/kie-wb-common-ala-openshift/kie-wb-common-ala-openshift-provider/src/main/java/org/guvnor/ala/openshift/config/impl/OpenShiftProviderConfigImpl.java [hunk 1]
```diff
@@ -581,7 +581,6 @@ public String toString() {
                 ", httpProxy=" + httpProxy +
                 ", httpsProxy=" + httpsProxy +
                 ", kubernetesApiVersion=" + kubernetesApiVersion +
-                ", kubernetesAuthBasicPassword=" + kubernetesAuthBasicPassword +
```
Comment (spolti) [inline kie-wb-common-ala/kie-wb-common-ala-openshift/kie-wb-common-ala-openshift-provider/src/main/java/org/guvnor/ala/openshift/config/impl/OpenShiftProviderConfigImpl.java:584]: isn't better instead remove, replace with ***?

Comment (Rikkola) [reply kie-wb-common-ala/kie-wb-common-ala-openshift/kie-wb-common-ala-openshift-provider/src/main/java/org/guvnor/ala/openshift/config/impl/OpenShiftProviderConfigImpl.java:584]: @spolti But what if the password is ***?

Ok I can do that :)

Comment (mareknovotny) [reply kie-wb-common-ala/kie-wb-common-ala-openshift/kie-wb-common-ala-openshift-provider/src/main/java/org/guvnor/ala/openshift/config/impl/OpenShiftProviderConfigImpl.java:584]: does it make sense? what if the `kubernetesAuthBasicPassword` is empty or not set at all? Would the string value still be presented as asterisks?

**Comment by Rikkola** [comment]
@spolti Like this?

**Comment by spolti** [comment]
> @spolti Like this?

I think so :)

**Comment by Rikkola** [comment]
@mareknovotny I think we can merge now.

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by Rikkola** [comment]
```
[2022-08-25T09:28:46.367Z] [ERROR] Failed to execute goal org.springframework.boot:spring-boot-maven-plugin:2.5.12:repackage (default) on project keycloak-kie-server-spring-boot-sample: Execution default of goal org.springframework.boot:spring-boot-maven-plugin:2.5.12:repackage failed. ConcurrentModificationException -> [Help 1]
[2022-08-25T09:28:46.367Z] org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.springframework.boot:spring-boot-maven-plugin:2.5.12:repackage (default) on project keycloak-kie-server-spring-boot-sample: Execution default of goal org.springframework.boot:spring-boot-maven-plugin:2.5.12:repackage failed.
```
jenkins do fdb

**Comment by Rikkola** [comment]
@mareknovotny It passed :)

**Pull request merged**